build.gradle

```diff
@@ -42,9 +42,12 @@ dependencies {
 
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compile "com.corundumstudio.socketio:netty-socketio:1.7.7"
+
+    compile group: 'org.json', name: 'json', version: '20160212'
```

Comment:
maybe use Moshi, which is already used as dep by the API (decreases file size and download time)